generatebundlefile/scripts/generate_bundle.sh

```diff
@@ -52,6 +53,6 @@ ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
 
 make oras-install
 
-ECR_PASSWORD=$(aws ecr-public get-login-password --region us-east-1 | tr -d '\n')
+ECR_PASSWORD=$(aws ecr-public get-login-password --region us-east-1)
 cd output/
-oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1" bundle-1.20.yaml
\ No newline at end of file
+${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1" bundle-1.20.yaml
```

Comment:
Nit: This exposes the password to anyone running `ps` at the time. Not the end of the world, but not a good habit. Consider instead:
```sh
aws ecr-public get-login-password --region us-east-1 \
    | oras login --username AWS --password-stdin "${IMAGE_REGISTRY}"
oras push "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1" bundle-1.20.yaml
```
The reason for the separate `login` command is that the `push` command doesn't accept `--password-stdin` sadly, though I imagine that will be added at some point.

This way, the password is passed via stdin, and isn't present as a program argument for anyone to see via the `ps` command.

Reply:
This is only run in Codebuild, I tried doing something similar to up above but wasn't able to get it working due to some docker wackiness. I can try again probably in another PR though.